src/api/routers/keyPairRouter.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+
+import { getKeyPairsList } from '../services/adminServiceClient';
+
+export function createKeyPairRouter() {
+  const keyPairRouter = express.Router();
+
+  keyPairRouter.get('/list', async (req, res) => {
+    const allKeyPairsPromise = getKeyPairsList();
+    const allKeyPairs = await Promise.resolve(allKeyPairsPromise);
```

Comment:
nit: any reason we can't just use `const allKeyPairs = await getKeyPairsList();` instead of these two lines?

Reply:
I changed it. thanks.